cardano-cli/test/cli/address-key-hash/run

```diff
@@ -12,12 +12,15 @@ setup_data_dir "${testname}"
 
 error=0
 
+assert_file_exists "${DATA_DIR}/address.vkey"
+
 ${CARDANO_CLI} shelley address key-hash \
     --verification-key-file "${DATA_DIR}/address.vkey" \
     > "${TEST_DIR}/vkey-hash"
 
 assert_file_exists "${TEST_DIR}/vkey-hash"
-
 assert_line_count 1 "${TEST_DIR}/vkey-hash"
+assert_string "f8d336c2e57d6f675bbe48c88584d8f4ee199b76676526c3b9fadb1d1a7b7ebb" "${TEST_DIR}/vkey-hash"
```

Comment:
If there is error in the file name argument this will still pass, file with error message will be created that's why I added this check.